resource_catalogue_fastapi/__init__.py

```diff
@@ -129,10 +161,222 @@ class ItemRequest(BaseModel):
     extra_data: Optional[Dict[str, Any]] = Field(default_factory=dict)
 
 
-class OrderRequest(ItemRequest):
-    product_bundle: str
+class ProductBundle(str, Enum):
+    general_use = "General use"
+    visual = "Visual"
+    basic = "Basic"
+    analytic = "Analytic"
+
+
+class ProductBundleRadar(str, Enum):
+    SSC = "SSC"
+    MGD = "MGD"
+    GEC = "GEC"
+    EEC = "EEC"
+
+
+class LicenceRadar(str, Enum):
+    SINGLE_USER = "Single User Licence"
```

Comment:
```suggestion
    SINGLE = "Single User Licence"
```

Suggestion rather than necessary but the multi options don't have user in the name

Reply:
Done